src/bin/uhyve.rs

```diff
@@ -274,9 +272,38 @@ impl CpuArgs {
 	}
 }
 
+#[derive(Parser, Debug)]
+struct GuestArgs {
+	/// The kernel to execute
+	#[clap(value_parser)]
+	kernel: PathBuf,
+
+	/// Arguments to forward to the kernel
+	kernel_args: Vec<String>,
+
+	/// Environment variables of the guest as env=value paths
+	///
+	/// `-e host` passes all variables of the parent process to the kernel (discarding any other passed environment variables).
+	///
+	/// Example: --env_vars ASDF=jlk -e TERM=uhyveterm2000
+	#[clap(short, long)]
+	env_vars: Vec<String>,
+}
+
 impl From<Args> for Params {
```

Comment:
Sidenote/Future work: Once we implement another structure for configuration files, we should implement a similar `From` trait and then clean up with a proper way to merge them.

I'm just treading carefully so as to not have to pay the price for it later.